src/middleware/filterMongooseExists.js

```diff
@@ -33,10 +33,10 @@ module.exports = (router) => (settings) => function(req, res, next) {
     query[settings.field] = exists;
   }
 
-  req.modelQuery = req.modelQuery || req.model || req.submissionModel || this.model;
```

Comment:
`req.modelQuery` and `req.Model` appear in OSS formio a lot. Doesn't this risk an OSS deployment breaking because the hook will be called and it will immediately fall back to`this.model` even if the modelQuery already exists on the request object?